build.sh

```diff
@@ -195,7 +195,7 @@ patch_osbuild() {
    ## in the places delivered by the RPM.
    mv /usr/lib/osbuild/tools/osbuild-mpp /usr/bin/osbuild-mpp
    mv /usr/lib/osbuild/osbuild /usr/lib/python3.13/site-packages/osbuild
-   mkdir /usr/lib/osbuild/osbuild
+   # mkdir /usr/lib/osbuild/osbuild
```

Comment:
this look like a leftover ?

Reply:
No, the only the `mkdir` cmd fails as the directory already exists. I should probably add `-p`